tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -320,6 +320,21 @@ def testee(inp: gtx.Field[[KDim], float]) -> gtx.Field[[KDim], float]:
     cases.verify(cartesian_case, testee, inp, out=out, ref=expected)
 
 
+def test_single_field_value(cartesian_case):
+    @gtx.field_operator
+    def testee_fo(a: cases.IKField) -> cases.IKField:
+        return a
+
+    @gtx.program
+    def testee_prog(a: cases.IKField):
+        testee_fo(a, out=a[1:2, 3:4])
+
+    a = cases.allocate(cartesian_case, testee_prog, "a")()
+    ref = np.asarray(a.asnumpy()[1, 3])
```

Comment:
Just a question, can't we directly slice the field and then extract the data as a numpy ndarray or would it fail?
```suggestion
    ref = a[1, 3].asnumpy()
```